solarforecastarbiter/io/nwp.py

```diff
@@ -1,10 +1,19 @@
 from pathlib import Path
 
+
 import numpy as np
 import xarray as xr
 
+
 BASE_PATH = ''
 
+
+def set_base_path(new_path):
+    global BASE_PATH
+    if new_path is not None:
```

Comment:
is there code that normally calls this function with `new_path = None`?

Reply:
Yes, ``cli.referencenwp`` calls ``set_base_path(nwp_directory)``. nwp_directory is not a required argument so it is None if not set